defaults/main.yml

```diff
@@ -7,3 +7,15 @@ tideways_download_folder_name: php-profiler-extension-master
 # If you use the Tideways UI, set this variable to your API key. Otherwise the
 # extension can be used along with the XHProf UI to view profiles.
 tideways_api_key: ''
+
+# XHProf UI
+tideways_install_xhprof_ui: yes
+
+# Use the commented URL and folder below if you want to use the 'official'
+xhprof_download_url: https://github.com/RustJason/xhprof/archive/php7.zip
+# xhprof_download_url: https://github.com/phacility/xhprof/archive/master.tar.gz
+xhprof_download_folder_name: xhprof-php7
```

Comment:
@geerlingguy how should these be set? `tideway_xhprof_download_url` or should we maybe reuse the `xhprof_download_url` variable with a conditional set_fact and `__tideways_xhprof_download_url`?